src/shogun/lib/SGMatrix.cpp

```diff
@@ -39,7 +41,9 @@ SGMatrix<T>::SGMatrix(bool ref_counting) : SGReferencedData(ref_counting)
 template <class T>
 SGMatrix<T>::SGMatrix(T* m, index_t nrows, index_t ncols, bool ref_counting)
 	: SGReferencedData(ref_counting), matrix(m),
-	num_rows(nrows), num_cols(ncols) { }
+	num_rows(nrows), num_cols(ncols)
```

Comment:
whitespace changes to separate commit pls
The patch is very large already and this makes it harder to parse